app/reducers/deviceReducer.js

```diff
@@ -7,15 +7,14 @@ import {
   SET_RAW_OBSERVABLE,
   DEVICE_CLEANUP
 } from "../epics/deviceEpics";
-import {
-  ActionType
-} from "../constants/interfaces";
+import { DEVICES } from "../constants/constants";
+import { ActionType } from "../constants/interfaces";
 
 export interface DeviceStateType {
```

Comment:
Fixed `null is not null or undefined` errors by making all reducer interface types covariant (read only with `+` annotation)